apps/fluent-tester/src/TestComponents/Shadow/ShadowTestBackgroundColor.tsx

```diff
@@ -0,0 +1,16 @@
+import { Appearance, Platform } from 'react-native';
+import { Theme } from '@fluentui-react-native/theme-types';
+
+/**
+ * The default dark mode iOS background color from the theme is black ('#000000'), which shadows are not visible against.
+ * Shadows are also not very visible when placed on a view that is black.
+ *
+ * For dark mode iOS, returns a dark grey that shadows are visible on. Otherwise, will return either undefined or the default
+ * background color from the theme based on whether or not a theme has been provided.
+ */
+export const shadowTestPageBackgroundColor = (t?: Theme) => {
+  if (Platform.OS === 'ios' && Appearance.getColorScheme() === 'dark') {
+    return '#313131';
+  }
+  return t?.colors.background;
+};
```

Comment:
This is pretty close to [building your own hook](https://reactjs.org/docs/hooks-custom.html), it would just need to switch from `getColorScheme` to `useColorScheme`. If we did that change, then this method should begin with `use`, e.g: `useShadowBackgroundColor`. However, I think something like this would be better in a themed style sheet, where you can just read the theme's apprearance rather than using `Appearance.getColorScheme()`

Reply:
Switched to themedStyleSheet